src/store/consts/static.ts

```diff
@@ -363,10 +363,10 @@ export const SIGNING_SNACKBAR_CONFIG: Omit<ISnackbar, 'open'> = {
 }
 
 export const ADDRESSES_TO_REVERT_TOKEN_PAIRS: string[] = [
```

Comment:
Here is the complete and accurate list of denominator tokens:
https://github.com/invariant-labs/stats/blob/e112ad3cefa7fe1d0d43109eef294e79924d0f80/api/solana/tickers_sorted.ts#L106